src/components/Icons.jsx

```diff
@@ -4,6 +4,7 @@ import Add from '../../sass/icons/chart settings/zoom-in/ic-zoomin-light.svg';
 import AddThin from '../../sass/icons/add/ic-add.svg';
 import Candle from '../../sass/icons/chart settings/chart types/candle/ic-candle-normal.svg';
 import Close from '../../sass/icons/close/ic-close.svg';
+import ClosedMarket from '../../sass/icons/close/ic-market-closed-line.svg';
 import Comparison from '../../sass/icons/chart settings/comparison/ic-comparison-normal.svg';
```

Comment:
where is `ic-market-closed-line.svg`? I didn't find it in my source code. Also I think you could just css for this. It's much easier to change later on.

Reply:
yeah you are right, I forgot to add the SVG file in the push.
I try to follow the prev codes, but I also can attached it as style icon only on css.
which one you prefer ?